src/group-signing-panacea-client.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { panacead, pendingWithoutPanacead } from "./testutils";
```

Comment:
@youngjoon-lee 
It's simple question.
This file used to test code?
Because it looks like import `testutils`

Reply:
Ya. The `*.spec.ts` is a convention of the test filename in Typescript :)

Reply:
In Javascript/Typescript, Jest is an equivalent of JUnit in Java.

We should configure `jest.config.js` like below:
https://github.com/medibloc/panacea-js/pull/40/files#diff-1e058ca1442e46581b13571fb8d261f9e1f5657e26c96634d4c1072f0f0347f1R5

Reply:
OK. I understand!